src/signature.c

```diff
@@ -70,7 +70,7 @@ bool initialize_signature(void)
 	struct tm *alttime;
 	struct stat statt;
 
-	string_or_die(&CERTNAME, "%s/%s", cert_path, SWUPDCERT);
+	string_or_die(&CERTNAME, "%s", cert_path);
```

Comment:
I'm not following.

SWUPDCERT is set by `--with-swupdcert`, so by removing the one place the macro is used, the option is no longer useful.